File: kubwa-annotations/src/main/java/com/wesleyelliott/kubwa/rule/TaxNumberRule.java
package com.wesleyelliott.kubwa.rule;

/**
 * Created by wesley on 2016/08/04.
 */

public class TaxNumberRule extends LuhnRule {

    public TaxNumberRule() {
        super(10);
    }

    @Override
    public boolean isValid(String value) {
        return validate(value);
    }
}

File: kubwa-annotations/src/main/java/com/wesleyelliott/kubwa/rule/TaxNumberRule.kt
package com.wesleyelliott.kubwa.rule

/**
 * Created by wesley on 2016/08/04.
 */

class TaxNumberRule : LuhnRule(10) {

    override fun isValid(value: String?): Boolean {
        return value != null && validate(value)
    }
}
